receiver/hostmetricsreceiver/config.go

```diff
@@ -25,6 +26,13 @@ type Config struct {
 	Scrapers                       map[string]internal.Config `mapstructure:"-"`
 	// RootPath is the host's root directory (linux only).
 	RootPath string `mapstructure:"root_path"`
+
+	// Collection interval for metadata.
+	// Metadata of the particular entity in the cluster is collected when the entity changes.
```

Comment:
Nit: I think the reference to "cluster" is not relevant to hosts.
```suggestion
	// Metadata of the particular entity is collected when the entity changes.
```